adapters/json_parse.go

```diff
@@ -125,3 +127,17 @@ func isArray(js *simplejson.Json, key string) bool {
 	}
 	return true
 }
+
+type jsonPath []string
+
+func (jp *jsonPath) UnmarshalJSON(b []byte) error {
+	strs := []string{}
+	var err error
+	if utils.IsQuoted(b) {
```

Comment:
Under what circumstances would the incoming string not be quoted?

Reply:
If it is an array, the first character won't be quotes.